src/core/task/Task.ts

```diff
@@ -1497,6 +1549,7 @@ export class Task extends EventEmitter<ClineEvents> {
 				this.isStreaming = false
 			}
 
+			/* kilocode_change: this was moved to the drainStreamInBackground() function above
```

Comment:
I would like commenting with // better; if Roo changes lines here I think we want to notice (and move it to the drainStreamInBackground?